mlir_graphblas/src/include/GraphBLAS/GraphBLASOps.td

```diff
@@ -777,6 +777,40 @@ def GraphBLAS_CommentOp : GraphBLAS_Op<"comment", []> {
     }];
 }
 
+def GraphBLAS_PrintOp : GraphBLAS_Op<"print", []> {
+    let summary = "graphblas print operation";
+    let description = [{
+        `graphblas.print` is used to pretty print values
+        to stdout.
+        This is intended to be used for debugging only.
+        The strings attribute is a list of strings.
+        This op is variadic and takes an arbitrary number of inputs.
+        The printing alternates between printing the strings
+        and the input values. 
+
+        ```mlir 
+        %c9_9_f32 = constant 9.9 : f32
+        %c1_i32 = constant 1 : i32
+        // prints "start 9.9 middle 1 end".
+        graphblas.print %c9_9_f32, %c1_i32 { strings = ["start ", " middle ", " end"] } : f32, i32
+        // prints "start 9.9 middle end zyx".
+        graphblas.print %c9_9_f32 { strings = ["start ", " middle ", " end", " z", "y", "x"] } : f32
+        // prints "start 9.91111".
```

Comment:
Can you update the description so it matches the new behavior? The two `// prints` statements above need to be fixed.

Reply:
https://github.com/metagraph-dev/mlir-graphblas/commit/49a1756d06cd5c71450b9aa92a474839ef972599 implements this suggestion.